src/getting-started/use-cases.md

```diff
@@ -0,0 +1,149 @@
+---
+shortTitle: Zilla Use Cases
+description: Zilla supports use cases as a service proxy for HTTP, SSE, MQTT, Kafka, and gRPC services and as an AsyncAPI Kafka gateway.
+breadcrumb: false
+---
+
+# Real-World Use Cases
+
+Zilla can be used as a service proxy (sidecar) or as an [AsyncAPI](https://www.asyncapi.com/) Kafka gateway.
+
+## Service Proxy
+
+Zilla enhances your existing services by adding metrics, logging, message validation, and authentication. Here's how it works for different service protocols:
+
+### For HTTP Services
+
+<div class="use_cases_cards">
+  <VPCard
+    logo="/assets/icons/proxy.svg"
+    title="HTTP Proxy Forwarding"
+    desc="A Zilla API gateway setup that forwards HTTP requests to an upstream server."
+    link="https://github.com/aklivity/zilla-examples/tree/main/http.proxy"
+  />
+  <VPCard
+    logo="/assets/icons/connecting kafka.svg"
+    title="HTTP to Kafka CRUD"
+    desc="A Zilla API gateway setup enabling CRUD operations over HTTP with Kafka integration."
+    link="https://github.com/aklivity/zilla-examples/tree/main/http.kafka.crud"
+  />
+</div>
+
+### For SSE Services
+
+<div class="use_cases_cards">
+  <VPCard
+    logo="/assets/icons/connecting kafka.svg"
+    title="SSE Kafka Fanout"
+    desc="A Zilla API gateway setup that distributes Kafka messages to multiple SSE clients."
+    link="https://github.com/aklivity/zilla-examples/tree/main/sse.kafka.fanout"
+  />
+  <VPCard
+    logo="/assets/icons/proxy.svg"
+    title="SSE Proxy with JWT"
+    desc="A Zilla API gateway setup that proxies SSE traffic with JWT authentication."
+    link="https://github.com/aklivity/zilla-examples/tree/main/sse.proxy.jwt"
+  />
+</div>
+
+### For MQTT Services
+
+<div class="use_cases_cards">
+  <VPCard
+    logo="/assets/icons/data governance.svg"
+    title="MQTT Kafka Broker"
+    desc="A Zilla API gateway setup acting as an MQTT broker with Kafka integration."
+    link="https://github.com/aklivity/zilla-examples/tree/main/mqtt.kafka.broker"
+  />
+  <VPCard
+    logo="/assets/icons/security.svg"
+    title="MQTT Kafka Broker with JWT"
+    desc="A Zilla API gateway setup that provides an MQTT broker with Kafka integration and JWT authentication."
+    link="https://github.com/aklivity/zilla-examples/tree/main/mqtt.kafka.broker.jwt"
+  />
+</div>
+
+### For Kafka Services
```

Comment:
For Kafka Services add all the solutions mentioned here: https://docs.aklivity.io/zilla/latest/solutions/concepts/kafka-proxies/